src/app/services/items/index.ts

```diff
@@ -1,9 +1,30 @@
+import { Observable, Subject } from 'rxjs';
+import { ItemsList } from '@core/pages/main/namespace';
 import ApiService from '../config';
 
+// for Saga usage
 export const getItemsService = (payload: { id: number }) => {
   return ApiService.get('/items', {
     id: payload.id,
   }).then((data: any) => {
     return data.data;
   });
 };
+
+// for Epic usage
+class ItemsService {
```

Comment:
Is it possible to keep services Promise based? And have rx/observable stuff outside? it will be better to have clean services without Observable context.

Also the task was about redux-observable so the middelare for redux actions,, so all the rx stuff should be inside middleware and not touch sourrunding context.